test/unit/model/serializer/test_serializer.py

```diff
@@ -165,7 +238,7 @@ def test_serializer_remove_old_models(self):
         # )
         # serializer = PersistentStorageSerializer(local_model_dir)
         # for _ in range(4):
-        #     serializer.save_model(model, pj, report=dummy_report)
+        #     serializer.save_model(model, self.pj, self.modelspecs, report=dummy_report)
```

Comment:
Can this commented code be removed?

Reply:
Can be used to train models to test if we can also delete them